raiden/tests/integration/test_recovery.py

```diff
@@ -292,7 +293,6 @@ def test_recovery_blockchain_events(
 
     app0.stop()
 
-    import gevent
```

Comment:
e163706302dafe6970b859f2fe3c84c34ad0138f fixed the same problem in a different way. This change exists only because of the rebase